### ISSUE TITLE
Debugging code

### ISSUE THREAD

**Issue by shigau97** [ravindurra/IT580Team4Lab4#4]

**Comment by shigau97** [comment]
For starters let's debug some code and try to find bugs as we did in our lab.

**Comment by shigau97** [comment]
Lets try to do in Quaver.shared as it has tons of variable and exceptions...

**Comment by ravindurra** [comment]
I tried to run debugger with it and I think "colorsplit" is doing some weird stuff.

**Comment by shigau97** [comment]
it is suppose to take value of a color but it stays on null,
![image](https://user-images.githubusercontent.com/59894266/117596866-f51be480-b109-11eb-97bf-52750b34c465.png)

**Comment by ravindurra** [comment]
yeah it has around 5 different values...lets open this and set a priority limit

**Comment by sg6162eq** [comment]
I ran a debugging on configmanager.cs and it seems to be okay.

**Comment by Michaelkalsi** [comment]
> I ran a debugging on configmanager.cs and it seems to be okay.

 I did it too....
